app/models/presenter/auction.rb

```diff
@@ -33,6 +25,10 @@ def max_allowed_bid
     delegate :bidder_name, :bidder_duns_number,
              to: :lowest_bid, prefix: :lowest
 
+    delegate :winning_bid, :veiled_bids, :single_bid?, :multi_bid?, :formatted_type,
```

Comment:
this is a small stylistic thing, but I find delegations and other long lists of attrs easiest to read when they are on their own lines and alphabetized, eg:

```ruby
delegate(
  :formatted_type
  :max_allowed_bid,
  :multi_bid?,
  :single_bid?,
  :user_can_bid?,
  :veiled_bids,
  :winning_bid,
  to: :auction_rules
)
```

what do you think?